components/ai_chat/resources/untrusted_conversation_frame/components/conversation_entries/index.tsx

```diff
@@ -69,6 +70,8 @@ function ConversationEntries() {
           const isEntryInProgress =
             isLastEntry && isAIAssistant && conversationContext.isGenerating
           const isHuman = turn.characterType === Mojom.CharacterType.HUMAN
+          const humanTurnSubmitted =
```

Comment:
maybe submitted isn't quite right? 
```suggestion
          const isGeneratingResponse =
```

also, can we not just rely on `conversationContext.isGenerating` here? Surely `isGenerating` isn't true while a person is typing?